.circleci/config.yml

```diff
@@ -3,44 +3,6 @@ version: 2.1
 aliases:
   - &working_directory ~/nodejs
 
-  - &restore_yarn_cache_node_18
-    keys:
-      - v1-yarn-cache-node_18-{{ .Branch }}-{{ checksum "yarn.lock" }}
-      - v1-yarn-cache-node_18-{{ .Branch }}
-      - v1-yarn-cache-node_18
-
-  - &save_yarn_cache_node_18
-    key: v1-yarn-cache-node_18-{{ .Branch }}-{{ checksum "yarn.lock" }}
-    paths:
-      - ~/.cache/yarn
-
-
-  - &restore_yarn_cache_node_20
-    keys:
-      - v1-yarn-cache-node_20-{{ .Branch }}-{{ checksum "yarn.lock" }}
-      - v1-yarn-cache-node_20-{{ .Branch }}
-      - v1-yarn-cache-node_20
-
-  - &save_yarn_cache_node_20
-    key: v1-yarn-cache-node_20-{{ .Branch }}-{{ checksum "yarn.lock" }}
-    paths:
-      - ~/.cache/yarn
-
-  - &restore_yarn_cache_node_22
-    keys:
-      - v1-yarn-cache-node_22-{{ .Branch }}-{{ checksum "yarn.lock" }}
-      - v1-yarn-cache-node_22-{{ .Branch }}
-      - v1-yarn-cache-node_22
-
-  - &save_yarn_cache_node_22
-    key: v1-yarn-cache-node_22-{{ .Branch }}-{{ checksum "yarn.lock" }}
-    paths:
-      - ~/.cache/yarn
-
-  - &yarn_install
-    name: Installing
-    command: yarn --frozen-lockfile
```

Comment:
We can simplify large parts of this:

1. We don't need a cache per Node.js version. Package managers don't install differently based on Node.js versions
2. We can migrate this into a command. That way it's native to CircleCI and validated by its language server also locally